plugins/header_rewrite/parser.cc

```diff
@@ -50,6 +52,12 @@ Parser::Parser(const std::string &line) : _cond(false), _empty(false)
         _tokens.push_back(std::string(1, line[i]));
       }
       continue; /* always eat whitespace */
+    } else if (line[i] == '\\') {
+      // erase a backslash in quoted-string
+      if (inquote && extracting_token) {
```

Comment:
My commit message might be not suitable. What I want to fix is we can't use **double quote** inside of quoted string.

We can make this more general, but it makes header_rewrite rules and parser code more complicated.
Do you have any specific use cases to escape something outside of quoted string?